### ISSUE TITLE
Multiple Anomalus' deathrattles cause surviving minions out of board

### ISSUE THREAD

**Issue by Kybxd** [HearthSim/hs-bugs#806]
* https://www.bilibili.com/video/av14468681/

**Comment by troggnostupidhs** [comment]
I'm pretty sure this is a duplicate of #517

**Comment by culinko** [comment]
I don't think this bug occurred at the time Anomalus was introduced, so this seems to be a new bug. However, we could add it to #517 and check if any of the initial bugs have been fixed in the meantime.

**Comment by Kybxd** [comment]
I also found a video in February 2017 showing this bug. And when he plays 'Brawl', there is no animations of out-of-board minions joining the brawl or being kicked out of the brawl. They do become invisible.
* http://www.bilibili.com/video/av8652207/

**Comment by troggnostupidhs** [comment]
https://youtu.be/xxmD2r3FSCM?t=6
